catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -31,7 +31,16 @@
                 name="Maximum Number Pending Results" id="maxPendingResults" required="true" type="Integer"
                 default="10000"
         />
-
+        <AD
+                description="Should the source library portion of the query be removed before execution."
+                name="Remove Source Library" id="removeSourceLibrary" required="true" type="Boolean"
+                default="true"
+        />
+        <AD
+                description="Library Version to return."
```

Comment:
Maybe a bit more descriptive - like "Library version returned when queried by external clients - indicates the version supported by this endpoint."